src/guides/v2.4/install-gde/system-requirements-tech.md

```diff
@@ -33,19 +33,19 @@ Upgrading the Magento applications and extensions you obtain from Magento Market
 
 ## Database
 
-MySQL 5.7.9 and 8.0
+*  MySQL 8.0 for on-premise installations
+*  MariaDB 10.4 for {{site.data.var.ece}} projects
 
-Magento is also compatible with MySQL NDB Cluster 7.4.*, MariaDB 10.2, 10.4 (recommended,) Percona 5.7, and other binary-compatible MySQL technologies.
+Magento is also compatible, but has not been tested and is not recommended, with MySQL 5.7.9, NDB Cluster 7.4.*, MariaDB 10.2, Percona 5.7, and other binary-compatible MySQL technologies.
```

Comment:
@jeff-matthews, as per previous discussion with Max Lesechko, we, probably, have to remove NDB Cluster and other binary-compartible technologies. 
Magento is also compatible, but has not been tested and is not recommended, with MySQL 5.7.9, MariaDB 10.2 and Percona 5.7.

Reply:
Done